src/state_manager/mod.rs

```diff
@@ -271,15 +271,8 @@ where
             crate::networks::NetworkChain::Calibnet => "calibrationnet",
             crate::networks::NetworkChain::Devnet(name) => name,
         }
-        .to_owned();
-
+        .to_string();
         Ok(name)
-        // let init_act = self
-        //     .get_actor(actor::init::ADDRESS, *st)?
-        //     .ok_or_else(|| Error::State("Init actor address could not be
-        // resolved".to_string()))?; let state =
-        // init::State::load(self.blockstore(), &init_act)?;
-        // Ok(state.into_network_name())
```

Comment:
Let's keep this or write a note about how the chain name should ideally be read from the genesis file. (Rahul is working on this.)